cmake/modules/falcosecurity-libs.cmake

```diff
@@ -27,8 +27,8 @@ else()
   # In case you want to test against another falcosecurity/libs version (or branch, or commit) just pass the variable -
   # ie., `cmake -DFALCOSECURITY_LIBS_VERSION=dev ..`
   if(NOT FALCOSECURITY_LIBS_VERSION)
-    set(FALCOSECURITY_LIBS_VERSION "0.7.0-alpha2")
-    set(FALCOSECURITY_LIBS_CHECKSUM "SHA256=4a62071043aa50cdd29016c7dc041b401f8fa551271ec7b3ba01a537e5eb8bce")
+    set(FALCOSECURITY_LIBS_VERSION "a3b4b5bd586b491d329529d827d50443f98b240a")
```

Comment:
I'd suggest tagging a version in libs before merging this PR.

@FedeDP wdyt?

Reply:
Makes sense! :+1: